adapters/handlers/rest/clusterapi/indices.go

```diff
@@ -358,8 +358,8 @@ func (i *indices) postObjectBatch(w http.ResponseWriter, r *http.Request,
 		return
 	}
 
-	index := i.db.GetIndexForIncoming(entschema.ClassName(class))
-	if index == nil {
+	index, err := i.db.GetIndexForIncoming(entschema.ClassName(class))
```

Comment:
Considering the point I mentioned previously, I recommend reverting this change. 
First, it appears unnecessary and are not related directly to RAFT. In addition, it could lead to conflicts with the main branch. Let's prioritise changes that are essential to minimise friction with the main branch. Feel free to propose this change again, once merge RAFT branch into main branch